### PR TITLE
Update cart show

### PR DESCRIPTION
**Pull request by amhursh**
Related Issue: #112 #97 

Summary: update cart

Author(s): amhursh

### CODE REVIEW

File: app/views/carts/show.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
           <th>Title</th>
           <th>Quantity</th>
           <th>Unit Price</th>
-          <th>Extended Price</th>
+          <th>Subtotal</th>
```
Comment (randy-darsh) [inline app/views/carts/show.html.erb:10]: yes, good

Comment (limsammy) [inline app/views/carts/show.html.erb:10]: Good call. Much more readable

File: app/views/carts/show.html.erb [hunk 2]
```diff
@@ -25,7 +25,7 @@
                 <% end %>
               </td>
               <td>$<%= item.formatted_price %> each</td>
-              <td>Item Subtotal: $<%= @cart.subtotal(item) %></td>
+              <td>$<%= @cart.subtotal(item) %></td>
```
Comment (randy-darsh) [inline app/views/carts/show.html.erb:28]: yes

**Review by limsammy — APPROVED** [review]
Looks good

**Pull request merged**